datumaro/util/test_utils.py

```diff
@@ -226,6 +217,20 @@ def test_save_and_load(test, source_dataset, converter, test_dir, importer,
         compare = compare_datasets
     compare(test, expected=target_dataset, actual=parsed_dataset, **kwargs)
 
+def compare_dirs(test, expected, actual, skip_empty_dirs=True):
```

Comment:
The `skip_empty_dirs` option doesn't seem to be used anywhere, so do we even need to have it?